### PR TITLE
Update application with changes to authorization middleware

### PR DESCRIPTION
**Pull request by ZachEddy**
# Update Authorization Middleware

The toolkit's authorization middleware has changed slightly (see [this pull request](https://github.com/infobloxopen/atlas-app-toolkit/pull/33) for a longer explanation). This pull request updates the contacts application to reflect these changes.

### CODE REVIEW

File: kube/pargs.yaml [hunk 1]
```diff
@@ -68,8 +68,8 @@ metadata:
   labels:
     pargset: authz
 operations:
-  - List
-  - Get
+  - Contacts.List
```
Comment (johnbelamaric) [inline kube/pargs.yaml]: I think we will need application as a criterion. See comments on other PR.

**Comment by ZachEddy** [comment]
Dan is about to release a new version of the toolkit, so let's hold off on merging this for now. We can move the contacts application to `toolkit v0.0.4` all at once, PARGs included.

File: cmd/server/main.go [hunk 2]
```diff
@@ -46,7 +53,7 @@ func main() {
 	if AuthzAddr != "" {
 		interceptors = append(interceptors,
 			// authorization interceptor
-			toolkit_auth.DefaultAuthInterceptor(AuthzAddr),
+			toolkit_auth.DefaultAuthInterceptor(AuthzAddr, applicationID),
```
Comment (kumaya) [inline cmd/server/main.go]: should be toolkit_auth.UnaryServerInterceptor();
also corresponding import should match latest version of toolkit

**Comment by ZachEddy** [comment]
Closing in favor of #18

**Pull request closed without merging**